src/sections/community/styles.scss

```diff
@@ -194,6 +338,103 @@
       }
     }
   }
+
+  &__navigator {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-left: auto;
+    margin-right: auto;
+    
+    // To help move the navigator all the way to the left, outside of normal max-width of parent
+    // 960px is the max width of the div above the <main> tag
+    @media (min-width: 960px) {
+      margin-left: calc(-1 * ((80vw - 960px) / 2));
+    }
+    
+    align-items: center;
+    width: 80vw;
+  }
+
+  &__image-grid {
+    display: grid;
+    height: 350px;
+    width: 90%;
+    gap: 0.1rem;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr; 
+    
+    // For some reason if the grid-template-columns are set to
+    // auto 17% auto auto auto auto; right side doesn't expand
+    // if all are auto nothing expands
+    // Code reviewer if you know please explain
+    grid-template-rows: 1fr 1fr;
+    overflow: hidden;
+    flex-grow: 2;
+    flex-shrink: 1;
+    transition: 300ms;
+
+    &:has(.pic1:hover) {
+      grid-template-columns: 1fr 1fr 1fr 0fr 0fr 0fr;
+      grid-template-rows: 1fr 0fr;
+      gap: 0rem;
+    }
+
+    &:has(.pic2:hover) {
+      grid-template-columns: 0fr 0fr 0fr 1fr 1fr 1fr;
+      grid-template-rows: 1fr 0fr;
+      gap: 0rem;
+    }
+
+    &:has(.pic3:hover) {
+      grid-template-columns: 1fr 1fr 0fr 0fr 0fr 0fr;
+      grid-template-rows: 0fr 1fr;
+      gap: 0rem;
+    }
+
+    &:has(.pic4:hover) {
+      grid-template-columns:  0fr 0fr 1fr 1fr 1fr 1fr;
+      grid-template-rows: 0fr 1fr;
+      gap: 0rem;
+    }
+
+    img {
+      object-fit: cover;
+      overflow: hidden;
+      border-radius: 20px;
+      width: 100%;
+      height: 100%;
+      margin-bottom: 0;
+    }
+
+    .pic1 {
+      grid-column: 1 / 4;
+      grid-row: 1;
+    }
+
+    .pic2 {
+      grid-column: 4 / -1;
+      grid-row: 1;
+    }
+
+    .pic3 {
+      grid-column: 1 / 3;
+      grid-row: 2;
+    }
+    
+    .pic4 {
+      grid-column: 3 / -1;
+      grid-row: 2;
+    }
+  }
+}
+
+.fullName {
+  color: rgba(0, 0, 0, 0.70);
+  font-family: "DM Sans";
+  font-size: 26px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
```

Comment:
i feel some of these aren't necessary, like `font-family` should already be DM Sans, etc. Having redundant properties would make it hard to do refactors if we, say, change the font in the future

Reply:
They should be gone, let me know if you see any more